lib/template/index.html

```diff
@@ -16,8 +16,8 @@
       repo: ''
     }
   </script>
-  <!-- Docsify v4 -->
-  <script src="//cdn.jsdelivr.net/npm/docsify@4"></script>
+  <!-- Docsify rc -->
+  <script src="//cdn.jsdelivr.net/npm/docsify@rc"></script>
   _plugins_
```

Comment:
Shall we make an option `-rc` for the `rc` version adoption instead of replace it as `rc` ?
By default we should always build and fetch the standard version like `@4` or `@5`.
And we don't need remove the `-rc` option when the `v5` released also,  cuz it will natural support new `rc` versions in future.

Reply:
Meanwhile, should we expose and option as `-v` also, by default fetch the standard version.
But user can use it like `-v 4.1`, `-v 5`, `-v rc` to replace the resources version of docsify.

Reply:
Should add-on this change directly or make another one？

Reply:
Check out a new branch from the current branch.

Reply:
> Feel free to submit a PR. Similar to plugin selection, you only need to provide the version selection and replacement content.

I think I had a mistake there, it seems the cli is designed to version specific which sync to docsify version.
Which means user can not use a v5+ cli to generate v4 resources.
I think it make sense to release the rc to match a specific version...
But if we have the `v6` rc, this cli of v5 version seems also work tho.